contributed_definitions/NXoptical_spectroscopy.nxdl.xml

```diff
@@ -400,6 +402,9 @@ TODO:
                         <item value="halogen lamp"/>
                         <item value="LED"/>
                         <item value="mercury cadmium telluride"/>
+                        <item value="deuterium lamp"/>
```

Comment:
Is this enumeration complete? If not maybe splitting up the concept into type(NXfabrication) and emitting medium(NXion/NXatom) is a better alternative?

Reply:
yea, indeed, I think this is better and will most probably eliminate some upcomming problems.

As I dont want to restructure this now, I added this to the TODO list in the yaml file.